storage/spider/spd_db_mysql.cc

```diff
@@ -2280,7 +2280,7 @@ int spider_db_mbase::fetch_and_print_warnings(struct tm *l_time)
       longlong res_num =
         (longlong) my_strtoll10(row[1], (char**) NULL, &error_num);
       DBUG_PRINT("info",("spider res_num=%lld", res_num));
-      my_printf_error((int) res_num, row[2], MYF(0));
+      my_printf_error((int) res_num, "%s", MYF(0), row[2]);
```

Comment:
Mentor @LinuxJedi says that, while it’s a security issue, the impact should be (relatively) insignificant.
We must address it nonetheless. I’ll probably extract this to a separate PR that follows the bug fix process (base on the oldest version in support).

Reply:
you don't need to increase the version tag.

Reply:
> I’ll probably extract this to a separate PR that follows the bug fix process (base on the oldest version in support).

1. 10.5: #3485
2. 10.6
3. 10.11